OxfordBodleian/Juel-Jensen/BDLaethf27.xml

```diff
@@ -410,6 +410,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     <desc>The name of the owner, mentioned in supplications throughout, was
                                         <persName ref="PRS13863WalattaE" role="owner"/>.</desc>
                                 </item>
+                                
+                                    <item xml:id="e4">
+                                       <!-- Miniatures 2–4 counted as one or two each?-->
+                                        <desc>Order of textual units and miniatures: Miniature 1; Column a: Textual
+                                        units I–IV, Textual unit V (beginning), Miniature 2; Textual
```

Comment:
According to how it was done in g 47, I guess this should be "Textual units I–V (beginning)".

But maybe it is less ambiguous if we list the textual units that are qualified with "(beginning)" or "(end)" separately, like here? In that case, g 47 should be changed.

(I'm open to either solution.)